cfpb/cfgov-refresh
# Ask CFPB: Spanish category page tag links

Add links on Spanish category pages to the tag pages for the category's top 10 tags 

## Additions

- Add top 10 tag links on Spanish category pages


## Testing

1. Check out branch
2. Navigate to [Spanish category page](http://localhost:8000/es/obtener-respuestas/categoria-comprar-un-vehiculo/) and verify `Temas Relacionados` tag links

## Screenshots
<img width="961" alt="screen shot 2017-06-13 at 7 42 25 am" src="https://user-images.githubusercontent.com/778171/27088202-defd2ba4-500b-11e7-99a8-5c395b9447c9.png">


## Notes

-


## Checklist
